src/polyglot-notebooks-vscode-common/src/extension.ts

```diff
@@ -70,13 +70,6 @@ const disposables: (() => void)[] = [];
 let surveryBanner: SurveyBanner;
 
 export async function activate(context: vscode.ExtensionContext) {
-    // Wait for ipynb extension to be ready, temporary.
-    const ipynbExtension = vscode.extensions.getExtension('vscode.ipynb');
-    if (ipynbExtension && !ipynbExtension.isActive) {
-        await ipynbExtension.activate();
-    }
-    metadataUtilities.setUseLegacyMetadata(ipynbExtension && !ipynbExtension.exports.dropCustomMetadata ? true : false);
```

Comment:
Its been 6 months since the `custom` property was dropped, 
I have removed this completely from VS Code without any support for fallbacks, its now time to remove this fallback from here as well.

Reply:
@colombod /cc